app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) \
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int,
+                 clean_power: int, average_rating: int,
+                 count_of_ratings: int) -> None:
+        self.distance_from_the_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(income, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        car.clean_mark = self.clean_power
+
+    def calculate_washing_price(self, car: Car) -> float:
+        clean_difference = self.clean_power - car.clean_mark
+        coefficient = self.average_rating / self.distance_from_the_city_center
+        single_car_wash_price = (
+            car.comfort_class * clean_difference * coefficient)
+        return round(single_car_wash_price, 1)
+
+    def rate_service(self, rating: int) -> None:
+        total_rating = self.average_rating * self.count_of_ratings + rating
+        self.count_of_ratings += 1
+        self.average_rating = total_rating / self.count_of_ratings
+        self.average_rating = round(self.average_rating, 1)
```

Comment:
Clean Code issue: you are calculating the average rating twice. You can avoid this by rounding the average rating only once after all calculations.